### PR TITLE
CSharp_oop_3 Hometask is completed

### PR DESCRIPTION
**Pull request by NadzeyaKiz**
CSharp_OOP_3 Homework is completed

### CODE REVIEW

File: Code/CSharpOOP3/Flowers.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
+﻿namespace CSharpOOP3
+{
+    public class Flowers  //Task_01
```
Comment (dnnyhlwdplc) [inline Code/CSharpOOP3/Flowers.cs]: just a minor comment. Class is single object, so it is better to call is Flower

File: Code/CSharpOOP3/Flowers.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    {
+        public string Color //(setter-only property)
+        {
+            set { Color = value; }
+        }
+
+        public double Price { get { return Price; } }  //(read-only property)
+
+        public string Name//(Task_03)
```
Comment (dnnyhlwdplc) [inline Code/CSharpOOP3/Flowers.cs]: you will get infinitive recursion here :)
please add field _name to achieve the result

File: Code/CSharpOOP3/Homework.cs [hunk 3]
```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSharpOOP3
+{
```
Comment (dnnyhlwdplc) [inline Code/CSharpOOP3/Homework.cs:2]: please remove unused classes from PR

File: Code/CSharpOOP3/Flowers.cs [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        {
+            get { return Name; }
+            private set { Name = value; }
+        }
+        public static string Type { get; set; }
+
+        public static double CalculateTotalPrice(Flowers[] flowersArray)// (Task_5)
```
Comment (dnnyhlwdplc) [inline Code/CSharpOOP3/Flowers.cs]: this method is not really connected with class Flowers :(

File: Code/CSharpOOP3/Helper.cs [hunk 5]
```diff
@@ -0,0 +1,32 @@
+﻿namespace CSharpOOP3
+{
+    public class Helper
```
Comment (dnnyhlwdplc) [inline Code/CSharpOOP3/Helper.cs]: this name is not really meaningful :(
there could be a lot of helpers in your project. Please rename to smth like FlowerHelper

File: Code/CSharpOOP3/Helper.cs [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+    {
```
Comment (dnnyhlwdplc) [inline Code/CSharpOOP3/Helper.cs]: hmm... I will set a mark to this task, but please review initial text. Helper should be connected to object you created - Flowers

**Review by dnnyhlwdplc — APPROVED** [review]

File: Code/CSharpOOP3/Flowers.cs (hunk 1 above)
Comment (NadzeyaKiz) [reply Code/CSharpOOP3/Flowers.cs]: done

File: Code/CSharpOOP3/Homework.cs (hunk 3 above)
Comment (NadzeyaKiz) [reply Code/CSharpOOP3/Homework.cs:2]: done

File: Code/CSharpOOP3/Helper.cs (hunk 5 above)
Comment (NadzeyaKiz) [reply Code/CSharpOOP3/Helper.cs]: done

File: Code/CSharpOOP3/Flowers.cs (hunk 2 above)
Comment (NadzeyaKiz) [reply Code/CSharpOOP3/Flowers.cs]: done

File: Code/CSharpOOP3/Helper.cs (hunk 6 above)
Comment (NadzeyaKiz) [reply Code/CSharpOOP3/Helper.cs]: done

File: Code/CSharpOOP3/Flowers.cs (hunk 4 above)
Comment (NadzeyaKiz) [reply Code/CSharpOOP3/Flowers.cs]: done

**Pull request merged**